cmd/kpod/tag.go

```diff
@@ -60,19 +60,19 @@ func addImageNames(store storage.Store, image *storage.Image, addNames []string)
 }
 
 func expandedTags(tags []string) ([]string, error) {
+	var labelName string
```

Comment:
Can this be moved inside the loop, as it's only used there?

Reply:
👍

Reply:
done